### PR TITLE
DT-2903 Added documentation explaining what the POC did and what we learnt from it

### PR DESCRIPTION
**Pull request by andymarke**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,3 +28,115 @@ This project provides a proof of concept around using GraphQL in spring boot as
 - We can talk to more than one endpoint for a given backend service and can be made performant
 - The service can be secured using HMPPS Auth
 - A possible working example is the DPS Offender Profile and the backend calls it currently makes
+
+## Proof of Concept results
+
+### Implementation
+
+- The server was implemented using Spring Boot and Spring GraphQL in ths project. This is viewable using the [GraphQL playground]("https://gql-api-dev.hmpps.service.justice.gov.uk/graphiql?path=/graphql)
```
Comment (neilmendum) [inline README.md:36]:
```suggestion
- The server was implemented using Spring Boot and Spring GraphQL in ths project. This is viewable using the [GraphQL playground](https://gql-api-dev.hmpps.service.justice.gov.uk/graphiql?path=/graphql)
```

**Review by edwardsneil — COMMENTED** [review]
Small markdown issue

File: README.md (hunk 1 above)
Comment (andymarke) [reply README.md]: Pushed a new version reming quotes around all links

**Review by edwardsneil — APPROVED** [review]
LGTM

**Pull request merged**